README.md

```
@@ -4,19 +4,17 @@ Automatically schedule tasks from github into google calendar, using Python
 
 ## Setup
 
-1. Clone this repository and create a virtual environment
+1. Clone this repository
 
 ```bash
 $ git clone
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)